### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by UlybinSergej**

### CODE REVIEW

File: task2/Task1.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        int[] array = new int[10];
+        Random random = new Random();
+        long sum = 0;
+        long minNum = Long.MAX_VALUE;
+
+        for (int i = 0; i < 10; i++) {
+            array[i] = random.nextInt();
+            sum += array[i];
+            minNum = (minNum > array[i]) ? array[i] : minNum;
+            System.out.println(array[i] + " ");
```
Comment (YanBelikov) [inline task2/Task1.java]: As I undarstand correctly you should to print all numbers in one line and separete it with space.
But here I see that you print one numper per cicle.

File: task2/Task3.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import java.util.Random;
+
+public class Task3 {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] positiveArray = new int[7];
+        int[] negativeArray = new int[7];
+        int[] array = new int[7];
+        int positiveCount = 7;
+        int negativeCount = 7;
```
Comment (YanBelikov) [inline task2/Task3.java]: You can merge this variables to one like "count"

File: task2/Task3.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package task2;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Task3 {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] positiveArray = new int[7];
```
Comment (YanBelikov) [inline task2/Task3.java]: use here and below new variable "count"

File: task2/Task1.java (hunk 1 above)
Comment (UlybinSergej) [reply task2/Task1.java]: Done

File: task2/Task3.java (hunk 3 above)
Comment (UlybinSergej) [reply task2/Task3.java]: Done

File: task2/Task3.java (hunk 2 above)
Comment (UlybinSergej) [reply task2/Task3.java]: Done

**Review by YanBelikov — APPROVED** [review]

**Pull request merged**